src/components/shortuct-bar/ShortcutBar.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { memo } from 'react';
+import { ShortcutButton } from '../shortuct-button/ShortcutButton';
+import { ShortcutButtonModel } from './ShortcutButtonModel';
+
+interface Props {
+  viewType: string;
+  shortcutButtons?: Array<ShortcutButtonModel>;
+  updateView: (date: Date | undefined) => void;
+}
+
+const getShortcutStyles = () => ({
+  root: {
+    display: 'flex',
+    padding: '2%',
+    gap: '1rem',
+    alignItems: 'center',
+    flexDirection: 'column' as const,
+    overflow: 'auto',
+    width: '120px',
+  },
+});
+
+function ShortcutBarComponent({ viewType, shortcutButtons, updateView }: Props) {
+  const shortcut = getShortcutStyles();
```

Comment:
I know we don't have a convention of naming style objects but we can create this object outside of the function to avoid creating a new instance of the same object for every render. 

We can directly just use it in the styles prop.

const shortcutStyles = {};

func () {
  <div styles={shortcutStyles} />
}

Reply:
Yes that's my bad. I was initially using this function to compute the style using the size property but later removed it. Must have forgotten to change the func to a const. Will update it.